src/components/navBar/navBar.js

```diff
@@ -206,58 +386,63 @@ MdNavBarController.prototype._initTabs = function() {
  * @param {string|undefined} oldValue Previous tab name.
  * @private
  */
-MdNavBarController.prototype._updateTabs = function(newValue, oldValue) {
-  var self = this;
-  var tabs = this._getTabs();
-  var oldIndex = -1;
-  var newIndex = -1;
-  var newTab = this._getTabByName(newValue);
-  var oldTab = this._getTabByName(oldValue);
-
-  if (oldTab) {
-    oldTab.setSelected(false);
-    oldIndex = tabs.indexOf(oldTab);
-  }
-
-  if (newTab) {
-    newTab.setSelected(true);
-    newIndex = tabs.indexOf(newTab);
-  }
-
-  this._$timeout(function() {
-    self._updateInkBarStyles(newTab, newIndex, oldIndex);
-  });
+MdNavBarController.prototype._updateTabs = function (newValue, oldValue) {
+    var self = this;
+    var tabs = this._getTabs();
+    var oldIndex = -1;
+    var newIndex = -1;
+    var newTab = this._getTabByName(newValue);
+    var oldTab = this._getTabByName(oldValue);
+
+    if (oldTab) {
+        oldTab.setSelected(false);
+        oldIndex = tabs.indexOf(oldTab);
+    }
+
+    if (newTab) {
+        newTab.setSelected(true);
+        newIndex = tabs.indexOf(newTab);
+    }
+
+    this._$timeout(function () {
+        self._updateInkBarStyles(newTab, newIndex, oldIndex);
+    });
 };
 
 /**
  * Repositions the ink bar to the selected tab.
  * @private
  */
-MdNavBarController.prototype._updateInkBarStyles = function(tab, newIndex, oldIndex) {
-  this._inkbar.toggleClass('_md-left', newIndex < oldIndex)
-      .toggleClass('_md-right', newIndex > oldIndex);
+MdNavBarController.prototype._updateInkBarStyles = function (tab, newIndex, oldIndex) {
+    this._inkbar.toggleClass('_md-left', newIndex < oldIndex)
+        .toggleClass('_md-right', newIndex > oldIndex);
 
-  this._inkbar.css({display: newIndex < 0 ? 'none' : ''});
+    this._inkbar.css({
+        display: newIndex < 0 ? 'none' : ''
+    });
 
-  if(tab){
-    var tabEl = tab.getButtonEl();
-    var left = tabEl.offsetLeft;
+    if (tab) {
+        var tabEl = tab.getButtonEl();
+        var left = tabEl.offsetLeft;
 
-    this._inkbar.css({left: left + 'px', width: tabEl.offsetWidth + 'px'});
-  }
+        this._inkbar.css({
+            left: left + 'px',
+            width: tabEl.offsetWidth + 'px'
+        });
+    }
 };
 
 /**
  * Returns an array of the current tabs.
  * @return {!Array<!NavItemController>}
  * @private
  */
-MdNavBarController.prototype._getTabs = function() {
-  var linkArray = Array.prototype.slice.call(
-      this._navBarEl.querySelectorAll('.md-nav-item'));
-  return linkArray.map(function(el) {
-    return angular.element(el).controller('mdNavItem')
-  });
+MdNavBarController.prototype._getTabs = function () {
+    var linkArray = Array.prototype.slice.call(
+        this._navBarEl.querySelectorAll('.md-nav-item'));
```

Comment:
Most of these lines aren't really changing. If you fix the indentation, I think the diff will look less overwhelming (and you won't be listed on every line to blame).